cdap-app-templates/cdap-etl/cdap-etl-api/src/main/java/co/cask/cdap/etl/api/StageContext.java

```diff
@@ -121,6 +143,15 @@
   Schema getOutputSchema();
 
   /**
+   * Return the output port schemas for the stage. The map key is the port name and the map value is the schema for
+   * that port. This map is the same map that was set at configure time by {@link MultiOutputPipelineConfigurable}.
+   * Stages that do not implement {@link MultiOutputPipelineConfigurable} should not use this method.
```

Comment:
whats the behavior when stages that do not implement MultiOutputPipelineConfigurable use this method? does it throw an exception?